### DISCUSSION TITLE
Limiting Number of multiple selections while selecting

### DISCUSSION THREAD

**Discussion by sanjeevdandin97** [PhilippeMarcMeyer/vanillaSelectBox#32] [Q&A] [answered]
Hello,
I have been using vanillaSelectBox for my project and I have to say it is pretty good library and satisfies whatever I need  with few touch ups and minor modifications.

Recently I was wondering if we can limit the selection of multiple select let's say like only allowing user to select 5 options from list which has 10 options. 
While declaring select box itself we can set limit to selection like below non working code:
```
 let mySelect = new vanillaSelectBox("#example",{ 
      limitSelection : 5
});
```
**P.S:** This is just an idea I got while working with this library.
Regards,
Dandin Sanjeev Rao

**Comment by PhilippeMarcMeyer** [comment]
That's a good idea !
I'd  like also to allow 2 levels with parent children system. What do you think about it ?

**Reply by sanjeevdandin97** [reply]
nice thought!. love the idea.
**p.s**: Sorry fore late reply

**Comment by PhilippeMarcMeyer** [comment]
i've got to work on this
just after the 2 levels drop down

**Comment by PhilippeMarcMeyer** [comment]
i havent give up your idea : I didnt had time. i wonder if i should hide the select/unselect all if a limit in the nr of selection is set ? What do you think ?

**Comment by PhilippeMarcMeyer** [comment, marked as answer]
I've done it : its in v 0.61 version :-)

**Reply by sanjeevdandin97** [reply]
Thanks! will try it!
